cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -409,6 +415,14 @@ body {
                     border-bottom: none;
                 }
             }
+
+            h3 {
+                margin-top: @grid_gutter-width / 2;
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/3147/commits/194f5f2c31c5342a26fe78c7342602083a7681a4:

<img width="529" alt="screen shot 2017-07-27 at 10 33 39 am" src="https://user-images.githubusercontent.com/778171/28686098-fbe618e0-72be-11e7-9a15-c84f3b6d4304.png">

Still need to look into the Sheerlike pages though.